include/deal.II/base/vectorization.h

```diff
@@ -1069,7 +1055,7 @@ class VectorizedArray<double, 8>
   }
 
   /**
-   * Load @p n_array_elements from memory into the calling class, starting at
+   * Load @p size() from memory into the calling class, starting at
```

Comment:
```suggestion
   * Load size() data items from memory into the calling class, starting at
```
same at all occurrences below similarly to the main class.